### PR TITLE
Allow translation of accessibility label string "Edit your profile"

### PR DESCRIPTION
**Pull request by t3chguy**

### CODE REVIEW

**Comment by t3chguy** [comment]
What is holding this up? It improves i18n.

File: react/features/toolbox/components/web/OverflowMenuProfileItem.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ import React, { Component } from 'react';
 import { Avatar } from '../../../base/avatar';
 import { getLocalParticipant } from '../../../base/participants';
 import { connect } from '../../../base/redux';
+import { translate } from '../../../base/i18n';
```
Comment (saghul) [inline react/features/toolbox/components/web/OverflowMenuProfileItem.js:8]: Please sort the import alphabetically by import path.

**Comment by saghul** [comment]
Jenkins please test this please.

**Comment by luixxiul** [comment]
@saghul would you please check? if lgtm, let's merge 😄

**Pull request merged**